spec/index.bs

```diff
@@ -4168,6 +4168,98 @@ enum GPUStoreOp {
 };
 </script>
 
+## Rasterization state ## {#render-pass-encoder-rasterization-state}
+
+The render pass encoder has several methods which affect how draw commands are rasterized to the
```

Comment:
we need to define what the default viewport state is

Reply:
And default scissor state too, if that's the case.

Reply:
Turns out the viewport/scissor defaults were already stated, but their formatting was a little unusual, which caused me to overlook it. For the sake of allowing cleaner reference from these algorithms I converted the existing definitions into internal slots. Let me know if there's concerns with that.